views/partials/_header.jade

```diff
@@ -0,0 +1,21 @@
+.page-row: .header
+  .nav-outer
+    .wrapper
+      .navbar#menu
+        a(href='/').logo: img(src='/img/logo-mama.png')
+        a(href='#menu').menu-btn.open: img(src='/img/menu-btn.png')
+        a(href='#').menu-btn.close: img(src='/img/menu-btn-open.png')
+        .menu-list
+          a(href='/')
+            span.home-text Home
+            img(src='/img/home-icon.png').home-icon
+          for category, i in navbar_features
+            a(href="/#{category_path}/#{slugify(category.title)}.html"
+              class="#{i + 1 === navbar_features.length ? 'last' : ''}")= category.title
+  .langs
+    .wrapper
+      - reg = new RegExp('\/' + 'lang' + '\/' + current_locale + '\/')
+      for locale in locales
+        - translated_path = _path.replace(reg, '/lang/' + locale.isocode + '/').replace('index.html', '')
```

Comment:
Have you remembered to reinstall NPM deps? I fixed that issue here (we use a fork of roots-records with my fix, which has been merged, but not published to NPM yet): https://github.com/declandewet/roots-records/commit/b7c909c2838f1f485783287fa31382a847e460c6